src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private static ArrayList<Task> todoList;
+
+    public TaskList() {
+        todoList = new ArrayList<>();
+    }
+
+    public static ArrayList<Task> getTodoList () {
+        return todoList;
+    }
+
+    public void add (Task task) {
```

Comment:
again, as with other lines of code, you could remove the space between the method name and parameter parentheses